server/game/cards/06_SEC/events/KreiasWhispers.ts

```diff
@@ -0,0 +1,42 @@
+import type { IAbilityHelper } from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import type { IEventAbilityRegistrar } from '../../../core/card/AbilityRegistrationInterfaces';
+import { GameStateChangeRequired, RelativePlayer, TargetMode, ZoneName } from '../../../core/Constants';
+
+export default class KreiasWhispers extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '3661373584',
+            internalName: 'kreias-whispers',
+        };
+    }
+
+    public override setupCardAbilities(registrar: IEventAbilityRegistrar, AbilityHelper: IAbilityHelper) {
+        registrar.setEventAbility({
+            title: 'Draw 3 cards, then put one card from your hand on the top of your deck and another card from your hand on the bottom of your deck',
+            immediateEffect: AbilityHelper.immediateEffects.sequential([
+                AbilityHelper.immediateEffects.draw({ amount: 3 }),
+                AbilityHelper.immediateEffects.selectCard({
```

Comment:
With the wording of this, I don't think they should all be sequential. The draw happens first, then the next two effects are simultaneous. I think we can probably combine a `then:` and a simultaneous system for this. You could even do multiple target resolvers for the top/bottom.